contrib_bots/lib/github_detail.py

```diff
@@ -0,0 +1,135 @@
+import re
+import os
+import sys
+import six.moves.configparser
+
+import requests
+
+from typing import Text, Dict
+
+
+class GithubHandler(object):
+    '''
+    This plugin provides details on github issues and pull requests when they're refrenced in the chat.
+    '''
+
+    config_path = os.path.expanduser('~/.contrib_bots/github_detail.ini')
+    owner = Text
+    repo = Text
+
+    def __init__(self):
+        config = six.moves.configparser.ConfigParser()
+        config.readfp(open(self.config_path))
+        if config.get('Github', 'repo'):
+            self.repo = config.get('Github', 'repo')
+            if config.get('Github', 'owner'):
+                self.owner = config.get('Github', 'owner')
+            else:
+                # Allowing undefined default repos would require multiple triage_message regexs.
+                # It's simpler to require them to be defined.
+                sys.exit('Default owner not defined')
+        else:
+            sys.exit('Default repo not defined')
+
+    def usage(self):
+        # type: () -> None
+        return ("This plugin displays details on github issues and pull requests. "
+                "To refrence an issue or pr type @github-detail then anytime in the message it's id like '#2561'."
```

Comment:
Done, I think I missed this because pep8 doesn't check docstrings for line length?